clients/banking/src/components/TransferWizardBeneficiary.tsx

```diff
@@ -86,87 +117,200 @@ export const TransferWizardBeneficiary = ({ initialBeneficiary, onSave }: Props)
+                  ),
+                },
+                ({ name, address }) => (
+                  <LakeAlert
+                    anchored={true}
+                    variant="neutral"
+                    title={t("transfer.new.bankInformation")}
+                  >
+                    <>
+                      <LakeText>{name}</LakeText>
+
+                      {match(address)
+                        .with(
+                          { addressLine1: P.string, postalCode: P.string, city: P.string },
+                          ({ addressLine1, postalCode, city }) => (
+                            <LakeText>
+                              {addressLine1}, {postalCode} {city}
+                            </LakeText>
+                          ),
+                        )
+                        .otherwise(() => null)}
+                    </>
+                  </LakeAlert>
+                ),
+              )
+              .with(
+                {
+                  beneficiaryVerification: AsyncData.P.Done(
+                    Result.P.Ok({ __typename: "BeneficiaryMatch" }),
+                  ),
+                },
+                () => (
+                  <LakeAlert
+                    anchored={true}
+                    variant="neutral"
+                    title={t("transfer.new.bankInformation")}
+                  >
+                    <LakeText>{t("transfer.new.nldValidBankInformation")}</LakeText>
+                  </LakeAlert>
+                ),
+              )
+              .with(
+                {
+                  beneficiaryVerification: AsyncData.P.Done(
+                    Result.P.Ok({
+                      __typename: "BeneficiaryMismatch",
+                      nameSuggestion: P.select(),
+                    }),
+                  ),
+                },
+                nameSuggestion => (
+                  <LakeAlert
+                    anchored={true}
+                    variant="neutral"
+                    title={t("transfer.new.bankInformation")}
+                  >
+                    {isNotNullish(nameSuggestion) ? (
+                      <LakeText>
+                        {t("transfer.new.nldInvalidBeneficiaryVerification.withSuggestion", {
+                          nameSuggestion,
+                        })}
+                      </LakeText>
+                    ) : (
+                      <LakeAlert
+                        anchored={true}
+                        variant="neutral"
+                        title={t("transfer.new.bankInformation")}
+                      >
+                        <LakeText>
+                          {t("transfer.new.nldInvalidBeneficiaryVerification.withoutSuggestion", {
+                            name: beneficiaryName.value,
+                          })}
+                        </LakeText>
+                      </LakeAlert>
+                    )}
+                  </LakeAlert>
```

Comment:
<img width="1051" alt="Screenshot 2024-03-14 at 11 36 41" src="https://github.com/swan-io/swan-partner-frontend/assets/1688645/6afc3442-fce8-43ce-a1fe-c8b3ca5683dc">

we have nested `LakeAlert` in here